rocks/RocksDatabase.java

```diff
@@ -267,4 +279,65 @@ public SchemaGraph schemaGraph() {
             return schemaGraph;
         }
     }
+
+    static class StatisticsBackgroundCounter {
+        private final RocksSession.Data session;
+        private final Thread thread;
+        private final Semaphore countJobNotifications;
+        private boolean isStopped;
+
+        StatisticsBackgroundCounter(RocksSession.Data session) {
+            this.session = session;
+            // NOTE: We use a Semaphore instead of CountDownLatch here because semaphore is closer to our usage pattern here.
```

Comment:
Let's remove this comment

Reply:
Oh I thought you want to put that in? @lolski

Reply:
I meant to add the comment in the PR, not in the code @lriuui0x0

Reply:
I'm adding it here: